### PR TITLE
TT-5980 / refactoring comments and tests

### PR DESCRIPTION
**Pull request by mativm02**
## Description

This PR modifies a comment to make it easier for the user to set an environment variable.
## Related Issue

https://tyktech.atlassian.net/browse/TT-5980
## Motivation and Context

Since the type of ports_whitelist is map[string]PortWhiteList, defining it through an environment variable was not feasible. In order to define it, a custom decoder is needed. 
When setting the above environment variable, we need to add a \ before each " character. If we don't do this, JSON decoder will fail because it can't recognize the fields.
## How This Has Been Tested

Tests weren't modified, only one line (in order to set the \ character before each " character)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
  - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit:  
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2925503344)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: ecd00743d7cbf304f1d466615a55d692d7eff58e 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2925507183)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: 932002e72b84d6ef3ec65e477974cf0ff7df0409 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2947126276)

File: config/config.go [hunk 1]
```diff
@@ -632,7 +632,7 @@ type Config struct {
 	// A policy can be defined in a file (Open Source installations) or from the same database as the Dashboard.
 	Policies PoliciesConfig `json:"policies"`
 
-	// Defines the ports that will be available for the API services to bind to in the following format: `{ “": “” }``.
+	// Defines the ports that will be available for the API services to bind to in the following format: `{“":“”}` (Remember to insert \ before the “ character).
```
Comment (tbuchaillot) [inline config/config.go:635]: @mativm02 I'd change to " "{“":“”}". Remember to escape JSON strings.

**Review by tbuchaillot — CHANGES_REQUESTED** [review]

**Review by tbuchaillot — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: ab0fc0ebc8812b2c1f78d0319bd18900e4875e73 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2954601637)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: 56a9fd9c606c762f46eb833644743c722c8bc938 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2954646729)

**Review by sredxny — APPROVED** [review]
lgtm

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: a97d353c4901cbcdae83dc4c4deff96cb32f9d4a 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3020920764)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4260/merge
Commit: 8ade8895c2060ae9cdbe5352d53f68fcd27c8547 
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3393976944)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: f2c600683692f67e6e40d68dc2ec3167a7f89c27 TT-5980 / refactoring comments and tests (#4260)

* refactoring comments and tests

* deleting testing print

* adding quotation marks to a comment

* modifying comment
Triggered by: push (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3394063978)

**Pull request merged**